homeassistant/components/ezviz/update.py

```diff
@@ -24,7 +24,7 @@
 
 UPDATE_ENTITY_TYPES = UpdateEntityDescription(
     key="version",
-    name="Firmware update",
+    translation_key="firmware",
```

Comment:
This should take its name from the device_class

Reply:
It doesn't, only number, sensor and binary sensor support this

Reply:
My mistake. It doesn't work for `update`